### ISSUE TITLE
please help 1728x1080 resolution, R301, Flatline, Rampage and Volt cannot work.

### ISSUE THREAD

**Issue by COLORFUL123311** [mgsweet/Apex-NoRecoil-2021#348]
please help 1728x1080 resolution, R301, Flatline, Rampage and Volt cannot work.

**Comment by BlackChaplin** [comment]
> please help 1728x1080 resolution, R301, Flatline, Rampage and Volt cannot work.

Please post screenshots with this weapon on your 1728x1080 resolution. I need to find the pixel coordinates at your resolution so that the script can correctly determine the equipped weapon.

**Comment by BlackChaplin** [comment]
You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS

**Comment by COLORFUL123311** [comment]
![20231207051045_1](https://github.com/mgsweet/Apex-NoRecoil-2021/assets/152357153/0098e49d-d36e-427b-874d-5d4bddc57b17)
![20231207051103_1](https://github.com/mgsweet/Apex-NoRecoil-2021/assets/152357153/9bb1cec2-8438-47ce-9bc4-cd2251c41d84)
![20231207051300_1](https://github.com/mgsweet/Apex-NoRecoil-2021/assets/152357153/66ce2d5f-b02d-44d3-94da-1d38ec33570b)
![20231207051426_1](https://github.com/mgsweet/Apex-NoRecoil-2021/assets/152357153/24809aa6-2c10-4e52-87c0-664a8b400553)

**Comment by COLORFUL123311** [comment]
> You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS

The guns in the picture are all unusable。。。Please helpT.T

**Comment by ggpeta** [comment]
If they work, I play at the same resolution and they work for me, the only weapons that don't work are the alternator and the volt. in 1728x1080.

**Comment by BlackChaplin** [comment]
> > You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS
> 
> The guns in the picture are all unusable。。。Please helpT.T

I added your resolution to my build. When starting, activate the voice checkbox, it will be very helpful for debugging and try to go to the training ground with a weapon. When selecting a weapon, if it is detected correctly, there should be a voice notification
[Apex-AimAssist-NoRecoil-2023-main.zip](https://github.com/mgsweet/Apex-NoRecoil-2021/files/13597993/Apex-AimAssist-NoRecoil-2023-main.zip)

**Comment by COLORFUL123311** [comment]
> > > You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS
> > 
> > 
> > The guns in the picture are all unusable。。。Please helpT.T
> 
> I added your resolution to my build. When starting, activate the voice checkbox, it will be very helpful for debugging and try to go to the training ground with a weapon. When selecting a weapon, if it is detected correctly, there should be a voice notification [Apex-AimAssist-NoRecoil-2023-main.zip](https://github.com/mgsweet/Apex-NoRecoil-2021/files/13597993/Apex-AimAssist-NoRecoil-2023-main.zip)

thanks ♥，i will try it later，♥X10

**Comment by COLORFUL123311** [comment]
> > > You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS
> > 
> > 
> > The guns in the picture are all unusable。。。Please helpT.T
> 
> I added your resolution to my build. When starting, activate the voice checkbox, it will be very helpful for debugging and try to go to the training ground with a weapon. When selecting a weapon, if it is detected correctly, there should be a voice notification [Apex-AimAssist-NoRecoil-2023-main.zip](https://github.com/mgsweet/Apex-NoRecoil-2021/files/13597993/Apex-AimAssist-NoRecoil-2023-main.zip)

amazing！IT can work ，thanks help me a lot~ ♥

**Comment by BlackChaplin** [comment]
> > > > You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS
> > > 
> > > 
> > > The guns in the picture are all unusable。。。Please helpT.T
> > 
> > 
> > I added your resolution to my build. When starting, activate the voice checkbox, it will be very helpful for debugging and try to go to the training ground with a weapon. When selecting a weapon, if it is detected correctly, there should be a voice notification [Apex-AimAssist-NoRecoil-2023-main.zip](https://github.com/mgsweet/Apex-NoRecoil-2021/files/13597993/Apex-AimAssist-NoRecoil-2023-main.zip)
> 
> amazing！IT can work ，thanks help me a lot~ ♥

No problem. If you wish, could you take screenshots of all the weapons that can be installed with GS? So that I can completely edit your resolution file and so that you can use additional functions.
If you want to help with this, please write to my github, the link to it is in the application

**Comment by COLORFUL123311** [comment]
> > > > > You can also send screenshots with a golden sight on the weapon from your list on which it can be installed, to correctly determine the use of GS
> > > > 
> > > > 
> > > > The guns in the picture are all unusable。。。Please helpT.T
> > > 
> > > 
> > > I added your resolution to my build. When starting, activate the voice checkbox, it will be very helpful for debugging and try to go to the training ground with a weapon. When selecting a weapon, if it is detected correctly, there should be a voice notification [Apex-AimAssist-NoRecoil-2023-main.zip](https://github.com/mgsweet/Apex-NoRecoil-2021/files/13597993/Apex-AimAssist-NoRecoil-2023-main.zip)
> > 
> > 
> > amazing！IT can work ，thanks help me a lot~ ♥
> 
> No problem. If you wish, could you take screenshots of all the weapons that can be installed with GS? So that I can completely edit your resolution file and so that you can use additional functions.

OK Please wait me back home ，I"m outside now

**Comment by hitch3ns** [comment]
@BlackChaplin 
![WhatsApp Image 2023-12-07 at 7 59 34 PM](https://github.com/mgsweet/Apex-NoRecoil-2021/assets/88315234/c8067521-039b-4c7b-9ecf-bbaae24bdb3c)
is it possible to make coordination better for alternator on 1680x1050 that's a screenshot

**Comment by POKONII** [comment]
Ensure that the color is # FFFFFF when the three points you take have a value of true

**Comment by BlackChaplin** [comment]
> Ensure that the color is # FFFFFF when the three points you take have a value of true

No, you can use dark/black pixel coordinates. And in this case, put the value "0" after the coordinates

**Comment by POKONII** [comment]
> > 确保颜色为 # FFFFFF 当您取的三个点的值为 true 时
> 
> 不可以，您可以使用深色/黑色像素坐标。在这种情况下，在坐标后放置值“0”

0不代表是dark，是非#FFFFFF，这个也可以在apexmaster.ahk里面改

**Comment by BlackChaplin** [comment]
> > > 确保颜色为 # FFFFFF 当您取的三个点的值为 true 时
> > 
> > 
> > 不可以，您可以使用深色/黑色像素坐标。在这种情况下，在坐标后放置值“0”
> 
> 0不代表是dark，是非#FFFFFF，这个也可以在apexmaster.ahk里面改

0 means that there will be no white color at these coordinates. And you don’t need to change anything in the apexmaster.ahk file
